code/modules/paperwork/pen.dm

```diff
@@ -48,13 +49,17 @@
 	if(!istype(M))
 		return
 
-	if(M.can_inject(user, 1))
-		user << "<span class='warning'>You stab [M] with the pen.</span>"
-		if(!stealth)
-			M << "<span class='danger'>You feel a tiny prick!</span>"
-		. = 1
+	if(!force)

```

Comment:
This is ugly, but idk how to do it in a non-ugly way. should i add an extra argument and then use that to skip it only in the edagger child? (e.g. call ..(M, user, stealth, extra) in edagger/attack(), and have extra just cause it to call ..())